.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 script:
   - export PYTHONPATH=`pwd`:$PYTHONPATH
   - python src/tests/runtests.py
-  # - nunit-console src/embed_tests/bin/x64/ReleaseMono/Python.EmbeddingTest.dll
+  # - nunit-console src/embed_tests/bin/Python.EmbeddingTest.dll
```

Comment:
On appveyor, when we run them on py3 they freeze and hang the testrunner until the build bot times out (30 minutes or so?).
I stress tested it on PY27 and its def fixed there. Ran 10 times no issues.